manifests/4.8/kube-descheduler-operator.crd.yaml

```diff
@@ -10,8 +12,6 @@ spec:
     plural: kubedeschedulers
     singular: kubedescheduler
   scope: Namespaced
-  subresources:
-    status: {}
```

Comment:
This top-level `subresources` field was removed, and replaced with one in each version below when I ran the generator... @ingvagabund will this work or is it the same problem you fixed in https://github.com/openshift/cluster-kube-descheduler-operator/pull/185?

Reply:
Yeah, it should work.

Reply:
Yeah, v1beta1 had it here, v1 has it per specific version.